scripts/histmakers/mw_with_mu_eta_pt.py

```diff
@@ -41,6 +45,11 @@
 
 era = args.era
 
+if args.IsoEfficiencysmoothing:
```

Comment:
I don't really understand the need to introduce this variable, as commented in the W script. IT just makes additionally lines that have to be maintained in multiple places (e.g., shouldn't it also be in the dilepton script?)

Reply:
ok in order to do this I need to modify the python scripts which initialize the C++ objects. It is also in the dilepton script, which is what you commented before

Reply:
I can do it if it is strictly needed

Reply:
Ok that variable was deleted, it is only used once in common.py

Reply:
> 

Ah ok sorry I understood now what you meant about the dilepton analysis